design/overview.md

```diff
@@ -81,43 +82,36 @@ And provide a security equivalent to or greater than IBC.
 
 ## Bootstraping
 
-This is based on [Installing Peggy on a live cosmos chain](notes.md#installing-peggy-on-a-live-cosmos-chain).
-We also assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
-as approaches to that are discussed in many other places. Here we focus on the *activation* step.
+We assume the act of upgrading the Cosmos-based binary to have peggy module is already complete,
+as approaches to that are discussed in many other places. Here we focus on the _activation_ step.
 
-Set up and register orchestrators:
+1. Each `Operator` generates an Ethereum and Cosmos private key for the `Orchestrator`. These addresses are signed and submitted by the Operators valoper key. The `Orchestrator` is now free to use these delegated keys for all Peggy messages.
+1. Each `Operator` configures their `Orchestrator` with the `Peggy Bridge ID` and `start threshold` they wish, their `Orchestrator` is to reject any proposals that do not agree on these values.
+1. Anyone submits a Peggy contract using a known codehash and the current validator set of the Cosmos zone to an Ethereum compatible blockchain. Note that step #1 must happen first because the `Orchestrator` Ethereum Addresses must be made known in order for verification to be possible.
```

Comment:
This is simpler than my idea :+1:

But "current validator set" is a misnomer. Maybe "MultiSig Set derived from the current validator set"? If I understand what you want to do.

Reply:
yes that's correct I'll make an edit there.